task4/APP/ansible.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- hosts: localhost
+  tasks:
+    - name: mvn build
+      command: sh build.sh
+    - name: git clone repo
```

Comment:
there are modules for everything in Ansible

ex : https://docs.ansible.com/ansible/latest/modules/git_module.html